src/plugins/release-issues/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @fileoverview Creates a new issue with the "release" label if the old one is closed
+ * @author Teddy Katz
+ */
+
+const moment = require("moment");
+
+const LABEL_NAME = "release";
+
+const ISSUE_TITLE_FORMAT = "[Scheduled release for ]MMMM Do, YYYY";
+
+const getIssueBody = (releaseDate) => `
+
+The scheduled release on ${releaseDate.format("dddd, MMMM Do, YYYY")} is assigned to:
+
+* (needs volunteers)
+* (needs volunteers)
+
+Please use this issue to document how the release went, any problems during the release, and anything the team might want to know about the release process. This issue should be closed after all patch releases have been completed (or there was no patch release needed).
+
+Resources:
+
+* [Release guidelines](https://eslint.org/docs/maintainer-guide/releases)
+
+`.trim();
+
+async function issueWasClosedMultipleTimes(github, { owner, repo, number }) {
+    const issueEvents = await github.issues.getEvents({
```

Comment:
I think you want to use pagination for this : https://probot.github.io/docs/pagination/
Not sure if we ever going to hit that scenario :smile:

Reply:
I agree with you that this would be a good idea, but it seems very unlikely that it will be a problem in practice, so I feel like it's not the best use of time to figure out how to fix it right now. In the worst-case scenario, a duplicate issue will end up being created under exceedingly rare circumstances, and we can just remove the label and close it.

Reply:
I agree with you.